### ISSUE TITLE
broken default parameter in PHP createProperties()

### ISSUE THREAD

**Issue by tirmog** [zeroc-ice/ice#1384]
We recently upgraded to ice-php-3.7.7 (from 3.7.1), due to needing to support php-8.1 in the very near future.
However, we ran into:
```
Warning: Ice\\createProperties() expects at least 1 parameter, 0 given in ...
```
Looks like it no longer defaults to empty array for the parameter?

**Comment by tirmog** [comment]
Looks like maybe the argument to zend_parse_parameters got changed inadvertently?
OLD (3.7.1): const_cast<char*>("|a!O!")
NEW (3.7.7): const_cast<char*>("|a!|O!")
(note the extra pipe)

**Comment by pepone** [comment]
Thanks @tirmog, I may be able to squeeze a fix before the upcoming 3.7.8 release
